silver/models/documents/base.py

```diff
@@ -343,6 +375,11 @@ def _entries(self):
     def transactions(self):
         return self.transaction_set.all()
 
+    @property
+    def transaction_total(self):
+        return (self.total * self.transaction_xe_rate if self.transaction_xe_rate
```

Comment:
Because it could leave to currency errors. `transaction_xe_rate` always exists if the document was `issued` and you can only create transactions for `issued` documents so theoretically you don't need `transaction_total` when `transaction_xe_rate` is not set.